### DISCUSSION TITLE
147

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#85] [General]
# 潮流周刊第147期 - 被关的马

封面图来源于周内在莫干山一度假中心里团建，里面有一个马厂，走过去看了看，很多高大威猛的马，很帅，可惜被关起来了。

https://weekly.tw93.fun/posts/147-%E8%A2%AB%E5%85%B3%E7%9A%84%E9%A9%AC/

**Comment by debuginn** [comment]
不错哦

**Comment by ChengWeiDong** [comment]
哈哈哈，原来你也在追乐夏😄

**Reply by tw93** [reply]
难得这么好看的综艺

**Comment by miluplay** [comment]
截图里面是什么浏览器呀

**Reply by debuginn** [reply]
Arc

**Reply by tw93** [reply]
哈哈 是edge，可以看我这个文章 https://tw93.fun/2023-08-20/edge.html

**Comment by YZQG** [comment]
棒棒棒！！！

**Comment by QuakeWang** [comment]
晚上一个人听《大梦》，又看了原版的 live，嚎啕大哭 出不来了。。。

**Reply by tw93** [reply]
很好听

**Comment by narwhrl** [comment]
图床的域名是alipayobjects.com欸，怎么调用的呀

**Reply by tw93** [reply]
抱歉 这个周刊其实是首先发在内网的，外部有离开的同学也想要看，在没有数据风险情况下，就同步到现在这里了
